particles/Tutorial/Kotlin/3_JsonStore/JsonStore.kt

```diff
@@ -0,0 +1,55 @@
+package arcs
+
+import kotlin.native.internal.ExportForCppRuntime
+
+/**
+ * Sample WASM Particle.
+ */
+class JsonStoreParticle : Particle() {
+
+  private val res = Singleton { PersonDetails() }
+  init {
+    registerHandle("inputData", res)
+  }
+
+  override fun populateModel(slotName: String, model: Map<String, String>): Map<String, String> {
+    if(res.get() != null) {
+      return model + mapOf(
+        "name" to res.get()!!.name,
```

Comment:
I recommend replacing this `if(/* nullcheck */) ... else ` logic with the elvis operator: 
https://kotlinlang.org/docs/reference/null-safety.html#elvis-operator
It'll be more succinct + kotlin conventional.